src/main/java/org/spongepowered/api/statistic/achievement/Achievement.java

```diff
@@ -75,89 +60,17 @@ static Builder builder() {
     Collection<Achievement> getChildren();
 
     /**
-     * Gets the {@link Statistic} which this achievement is backed by if it is available.
+     * Returns the {@link ItemStack} that this achievement is represented by.
      *
-     * @return The source statistic, if available
+     * @return ItemStack representation
      */
-    Optional<Statistic> getSourceStatistic();
+    Optional<ItemStack> getItemStack();
 
     /**
-     * If this achievement is backed by a statistic (eg. if
-     * {@link #getSourceStatistic()} is not absent) then this will return the
-     * target value which must be reached for this achievement to be awarded to
-     * a player.
+     * Returns true if this is a "special" achievement.
      *
-     * @return The target value, if available
-     */
-    Optional<Long> getStatisticTargetValue();
```

Comment:
As far as I could tell there is no relationship between achievements and "source" statistics? I could be wrong though. Also an achievements value is 0 for locked and >0 for unlocked although this is not exposed by the API.